compiler/optimizations/forallOptimizations.cpp

```diff
@@ -1911,13 +1940,20 @@ static bool handleYieldedArrayElementsInAssignment(CallExpr *call,
   DefExpr *checkSymDef = new DefExpr(checkSym, checkCall);
   forall->insertBefore(checkSymDef);
 
+  // if `onlyIfFastFollower` is set, we want to revert this flag to true for the
+  // fast follower copy, if not, we really don't care and pass true so that it
```

Comment:
Should this say "we want the flag to be false for the non-fast follower" ?

Reply:
That's right. I can add that to make it clear.

(I think this is one of the things where adding a new AST type to represent optimizations can really help, although I like having primitives to represent such optimizations, adding more and more bool flags to them gets ugly real fast)